### PR TITLE
fix: update bug template

### PR DESCRIPTION
**Pull request by ckrook**
Following up on our discussion yesterday and updating our bug template accordingly. Please read through and give your opinion.

## **Describe pull-request**  
Updating the bug template so that we can better understand the developers problem and help in a more efficient way.

## **How to test**  
Please read through the new bug template

### CODE REVIEW

**Review by nathalielindqvist — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+Code Example
+Please create a minimal, reproducible example in an online sandbox (e.g., CodeSandbox, StackBlitz, etc.). Providing code in a sandbox helps us understand and troubleshoot the issue faster.
+
```
Comment (theJohnnyMe) [inline .github/ISSUE_TEMPLATE/bug_report.md:25]:
```suggestion
Code Example
Please create a minimal, reproducible example in an online sandbox and paste a link to it here. We strongly recommend the usage of [Stackblitz](https://stackblitz.com/) as it offers ready-made templates for different frameworks and works great with Tegel. Providing code in a sandbox helps us understand and troubleshoot the issue faster.

Other sandbox tools: CodePen, CodeSandbox


```

Comment (theJohnnyMe) [reply .github/ISSUE_TEMPLATE/bug_report.md:25]: @ckrook  - I changed this comment as I think Stackblitz would work best for us and our users. I am thinking, how do we make a field where people can paste their links in?

Comment (ckrook) [reply .github/ISSUE_TEMPLATE/bug_report.md:25]: @theJohnnyMe  Sounds good to me!

**Review by theJohnnyMe — APPROVED** [review]
Great!

**Comment by timrombergjakobsson** [comment]
Should we merge this one?

**Pull request merged**